build/_build/Build.Steps.cs

```diff
@@ -915,13 +918,16 @@ projectPath switch
             // Annoyingly this rebuilds everything again and again.
             var targets = new [] { "RestoreSamplesForPackageVersionsOnly", "RestoreAndBuildSamplesForPackageVersionsOnly" };
 
+            // /nowarn:NU1608 - Detected package version outside of dependency constraint
+            // /nowarn:NU1701 - Package 'x' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.1'.
```

Comment:
Should we ignore this warning? Why are we restoring a net461 package into netcore2.1 app?

Reply:
> Why are we restoring a net461 package into netcore2.1 app

A bunch of packages don't support .NET Core:

*  MySql.Data 6.9.12
* StackExchange.Redis 1.0.488
* MongoDB.Driver 2.0.2
* RabbitMQ.Client 3.6.9

Thanks to the .NET 461 compat shim, they _can_ still be used with .NET Core however. This isn't something we can "fix", other than not testing those packages, but given this is a "valid" setup, I think we should be testing it. Hence ignoring the warnings.

> nowarn:NU1608 - Detected package version outside of dependency constraint

This is an issue with the ES NEST library, again when we're testing multiple different versions. We could maybe remove the JSON.NET lib entirely and just reference it transitively. I'll give that a try.

Reply:
Realised I added the nowarn to Samples.Elasticsearch.V5 directly, so we don't need this "global" nowarn I don't think

Reply:
Thanks for clarifying!